composer.json

```diff
@@ -75,9 +75,7 @@
       "AmpProject\\": "vendor/ampproject/amp-wp/lib/common/src",
       "AmpProject\\Optimizer\\": "vendor/ampproject/amp-wp/lib/optimizer/src"
     },
-    "files": [
-      "includes/polyfills/mbstring.php"
-    ]
+    "files": ["includes/polyfills/mbstring.php"]
```

Comment:
Please undo this unrelated and unwanted change

Reply:
I have to do it with `--no-verify`, as some lint-fix will otherwise just change it back. 🤷 

It'll happen again next time this file is touched by someone else and I have to merge it into a PR - then that same linter will rewrap the lines.

Reply:
Strange. I'lll take a look then. This should not happen.